pkg/lib/machine-dialogs.js

```diff
@@ -147,6 +149,7 @@
                 conn_options['temp-session'] = false; /* Compatiblity option */
                 conn_options['session'] = 'shared';
                 conn_options['host-key'] = machine.host_key;
+                conn_options['unknown-host'] = "1";
```

Comment:
Yes it is used for not localhost machines. We have a host-key but we need to preserve the fact we have permission to talk to this host. Normally that permission is only for private sessions. But in these cases it needs to be a public session but we can't write the key to disk. So the presence of this key signifies that it's ok to talk to the otherwise unknown host. Because we can't write the key to disk,

Does it need a rename to make it clearer?